TheInternet/HoversPage.cs

```diff
@@ -0,0 +1,51 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+
+namespace TheInternet;
+
+public class HoversPage : BasePage
+{
+    IList<IWebElement> userImagesList => driver.FindElements(By.CssSelector("div [class = 'figure']"));
```

Comment:
The CSS selector language has 4 top level.... things:

A 'raw name' (no idea what you'd call it), e.g. `button` which selects by tag name
A dot-prefixed word, e.g. `.btn-primary` which selects by class (whether the element has that class by any means)
A hash-prefixed word, e.g. `#submit-button` which selects by id
A bracket enclosed key-value pair, e.g.... this one.

It then has pseudo element selectors e.g. `:focus`, which selects based on the state of the element, usually as the result of a user interaction (more on that later)

The one you've chosen will indeed work, but it's a bit more verbose and more importantly it's brittle. The `=` comparison is a strict, string-based comparison. If another class is added to the element it will no longer match